expcore/Mini_games.lua

```diff
@@ -189,6 +197,10 @@ function Mini_games.stop_game()
     for i,command_name  in ipairs(mini_game.commands) do
         Commands.disable(command_name)
     end
+    for i,player in ipairs(game.connected_players) do
```

Comment:
I think game.players and game.connected_players require you to use pairs but I could be wrong about that.

Reply:
I am  sure your wrong here, as i tested it it does not use the event.player_index but it is an array.  game.connected_players["name"] is not even valid i also tested that.